.sql

```diff
@@ -0,0 +1,19 @@
+createdb twoiam 
+psql twoiam 
+
+drop table if exists users;
+drop table if exists images;
```

Comment:
You may want to omit this line as it makes it unsafe to run against an existing database if you want to keep the data.

Reply:
changes made